src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -105,10 +109,21 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long id = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
         Driver driver = new Driver();
         driver.setId(id);
         driver.setName(name);
         driver.setLicenseNumber(licenseNumber);
+        driver.setLogin(login);
+        driver.setPassword(password);
         return driver;
     }
+
+    @Override
+    public Optional<Driver> findByLogin(String login) {
+        return getAll().stream()
```

Comment:
don't use the getAll method, create a separate query that will be looking for a driver by login